package-lock.json

```diff
@@ -0,0 +1,3094 @@
+{
```

Comment:
we don't need to commit `package-lock.json`, right?

Reply:
I intentionally added `package-lock.json` so that it locks down the dependencies.